settings/base.py

```diff
@@ -130,6 +130,7 @@
     "blog",
     "classifiers",
     "core",
+    "favourites",
```

Comment:
Please change this and the rest of the app name from the British spelling of favourites to the American spelling of favorites. I'm happy to assist in the rename if that causes your database any issues.